### PR TITLE
Fixes #29 Screenshot should include all windows, not just the key window

### LINKED ISSUE #29: Screenshot should include all windows, not just the key window
This will make bug reporting all the more useful when using overlays and layered windows.

Apple has an example on how to composite a screenshot in UIKit: http://web.archive.org/web/20100725234258/http://developer.apple.com/iphone/library/qa/qa2010/qa1703.html (thanks @natan for the link!)

We'd likely want to just adopt that code in `logScreenshot`

### PR DESCRIPTION
**Pull request by natan**

### CODE REVIEW

File: Logging/ARKLogDistributor.m [hunk 1]
```diff
@@ -246,9 +246,11 @@ - (void)logScreenshot;
     UIImage *screenshot = nil;
     
     @try {
-        UIWindow *window = [[UIApplication sharedApplication] keyWindow];
-        UIGraphicsBeginImageContextWithOptions(window.bounds.size, YES, 0.0);
-        [window.layer renderInContext:UIGraphicsGetCurrentContext()];
+        CGRect const screenBounds = [UIScreen mainScreen].bounds;
+        UIGraphicsBeginImageContextWithOptions(screenBounds.size, NO, 0.0);
+        for (UIWindow *const window in [UIApplication sharedApplication].windows) {
+            [window drawViewHierarchyInRect:screenBounds afterScreenUpdates:NO];
```
Comment (dfed) [inline Logging/ARKLogDistributor.m]: `drawViewHierarchyInRect:afterScreenUpdates:` is iOS 7 and later. Will jump on your branch and make the necessary changes later tonight.

**Comment by dfed** [comment]
LGTM. @natan merge if you like my changes

**Comment by shawnwelch** [comment]
LGTM

**Comment by dfed** [comment]
Merging and rolling a new version.

**Pull request merged**